index.html

```diff
@@ -130,8 +131,288 @@
 
     <section>
       <h2>Introduction</h2>
-      <p class="issue" data-number="1"></p>
+      <p>
+This specification describes the Ecdsa Secp256k1 Signature created in 2019 for the
+Linked Data Signatures [[LD-SIGNATURES]] specification. It uses the RDF
+Dataset CANONICALIZATION Algorithm [[RDF-DATASET-CANONICALIZATION]] to transform the
+input document into its canonical form. It uses SHA-256 [[RFC6234]] as the
+<a>message digest algorithm</a> and ES256K JWS with Unencoded Payload Option defined in [[rfc7797]] as the <a>signature algorithm</a>.
```

Comment:
What's the background thinking behind using the unencoded payload option instead of base64URL encoding? I've been using the Base64URL solution, but I'm not sure if it will work with the LDSignatures concepts.

Reply:
its still base64url encoded it just does not copy the encoded payload into the jws.

thats why he signature looks like: HEADER..SIGNATURE instead of HEADER.PAYLOAD.SIGNATURE.

There is some more detail here: https://github.com/decentralized-identity/lds-ecdsa-secp256k1-2019.js/tree/master/packages/es256k-jws-ts#jws---json-ld-signature-details

I also thought this was strange, but I tried to follow Ed25519Signature2018 to the T...

https://github.com/decentralized-identity/lds-ecdsa-secp256k1-2019.js/blob/master/packages/lds-ecdsa-secp256k1-2019/src/__tests__/interop/jsonld-signatures.spec.ts#L101

Reply:
Awesome thanks for the details. I'll dig into the links you provided to learn more about it.